signer/src/error.rs

```diff
@@ -39,6 +39,10 @@ pub enum Error {
     #[error("transaction is missing, txid: {0}, block hash {1:?}")]
     BitcoinTxMissing(bitcoin::Txid, Option<bitcoin::BlockHash>),
 
+    /// to exist in bitcoin core but it does not.
```

Comment:
Think you missed some leading words here :)

Reply:
https://github.com/stacks-network/sbtc/pull/744/commits/1da998d1d1db6b789b67ee90d1ff08d57a170a0e